core/CMakeLists.txt

```diff
@@ -773,10 +773,10 @@ endif()
 
 include(BareosConfigureFile)
 
-add_subdirectory(scripts)
-add_subdirectory(manpages)
 add_subdirectory(platforms)
 add_subdirectory(src)
+add_subdirectory(scripts)
+add_subdirectory(manpages)
```

Comment:
I need to check for a target defined in `src` within `scripts` which is order-dependant.
Also `src` needs to see declarations from `platforms` so that is order-dependant, too.
So the best comment I could come up with is "it needs to be in that order because things depend on each-other".
But that is true for large portions of all `CMakeLists.txt`s so I don't think we can add a reasonable comment here.

Reply:
Hmm... i guess you are right.